tests/01_eradiate/01_unit/scenes/test_loader.py

```diff
@@ -0,0 +1,945 @@
+      "wavelengths": {
+        "values": [
+          442.95000000000005,
+          490.45000000000005,
+          560.4,
+          665.25,
+          681.55,
+          709.1,
+          754.15,
+          833.5,
+          865.6,
+          1242.5,
+          1610.5,
+          2120.0,
+          2199.0
+        ],
+        "units": "nanometer",
+        "type": "interpolated"
+      },
+      "values": [
+        0.882742,
+        0.897852,
+        0.907004,
+        0.910396,
+        0.913883,
+        0.902941,
+        0.900788,
+        0.856648,
+        0.848049,
+        0.368409,
+        0.04536,
+        0.016815,
+        0.053261
+      ],
+      "type": "interpolated"
+    }
+  },
+  "canopy": {
+    "type": "discrete_canopy",
+    "instanced_canopy_elements": [
+      {
+        "canopy_element": {
+          "type": "mesh_tree",
+          "id": "PIMO1",
+          "mesh_tree_elements": [
+            {
+              "id": "Wood",
+              "mesh_filename": "tree-species/Pinus-Montana/PIMO1/Pinus_mugo_PIMO1_-_Original/Wood.ply",
+              "reflectance": {
+                "quantity": "reflectance",
+                "wavelengths": {
+                  "values": [
+                    442.95000000000005,
+                    490.45000000000005,
+                    560.4,
+                    665.25,
+                    681.55,
+                    709.1,
+                    754.15,
+                    833.5,
+                    865.6,
+                    1242.5,
+                    1610.5,
+                    2120.0,
+                    2199.0
+                  ],
+                  "units": "nanometer",
+                  "type": "interpolated"
+                },
+                "values": [
+                  0.072764,
+                  0.082621,
+                  0.109325,
+                  0.157819,
+                  0.167218,
+                  0.200941,
+                  0.239725,
+                  0.303587,
+                  0.329257,
+                  0.525503,
+                  0.511729,
+                  0.383173,
+                  0.394486
+                ],
+                "type": "interpolated"
+              }
+            }
+          ]
+        },
+        "instance_positions": [
+          [
+            [
+              0.2813481616359169,
+              0.9596057586030265,
+              0.0,
+              18.5271
+            ],
+            [
+              -0.9596057586030265,
+              0.2813481616359169,
+              0.0,
+              97.4987
+            ],
+            [
+              0.0,
+              0.0,
+              1.0,
+              0.0
+            ],
+            [
+              0.0,
+              0.0,
+              0.0,
+              1.0
+            ]
+          ]
+        ]
+      }
+    ],
+    "size": [
+      103.1214,
+      103.2308,
+      15.0213
+    ]
+  }
+}
+"""
+
+
+def serialize_fields(parsed_scene: dict) -> dict:
```

Comment:
Maybe add a comment here to outline the role of this function in the test suite?